native/src/routes/Pois.tsx

```diff
@@ -60,6 +76,23 @@ const Pois = ({
 }: PropsType): ReactElement => {
   const { t } = useTranslation('pois')
   const theme = useTheme()
+  const [selectedFeature, setSelectedFeature] = useState<Feature<Point> | null>(null)
+  const [userLocation, setUserLocation] = useState<number[] | undefined>(undefined)
+  const [featureLocations, setFeatureLocations] = useState<Feature<Point>[]>(prepareFeatureLocations(pois))
+
+  useEffect(() => {
+    if (!path) {
```

Comment:
What is this check needed for?

Reply:
we don't need the hook if we are on the details page (if path is available)